ivy/functional/frontends/torch/reduction_ops.py

```diff
@@ -0,0 +1,33 @@
+import ivy
+
+
+def argmax(input, dim=None, keepdim=False):
```

Comment:
Hey @ahmedo42 , dim should not have a default value of `None` as mentioned in the docs [here](https://pytorch.org/docs/stable/generated/torch.argmax). Please review the arguments of the other functions as well to make sure they are consistent with torch's docs

Reply:
I think that I should keep that `dim=None` because if we don't, the user has to explicitly add it which is not the same behavior  in torch

Reply:
@ahmedo42 you seem to be right. Sry for the inconvenience but It seems to be a bug with torch's docs. Anyway, in that case, I agree we should add `dim=None` to all the functions. Thanks!